setup.py

```diff
@@ -14,6 +14,7 @@
         # cryptography does not use semver and allows breaking changes within `0.3` version bumps.
         "cryptography>=2.5,<2.7",
         "eth-hash>=0.1.4,<1",
+        "eth-keys>=0.3.0,<0.4.0",
```

Comment:
@jannikluhn Can you try again with

```suggestion
        "eth-keys>=0.2.3,<0.3.0",
```